### PR TITLE
tuimoji: disable test

### PR DESCRIPTION
**Pull request by Luciogi**
#### Testing the changes
- I tested the changes in this PR: **YES**
#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glib)

### CODE REVIEW

File: srcpkgs/tuimoji/template [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
 pkgname=tuimoji
 version=1.0.0
-revision=7
+revision=8
 build_style=python3-module
-pycompile_module="tuimoji"
+#pycompile_module="tuimoji"
 hostmakedepends="python3-setuptools"
 depends="python3-urwid xclip python3-setuptools"
+checkdepends="python3-pip"
```
Comment (classabbyamp) [inline srcpkgs/tuimoji/template:9]: this is probably wrong, something saying "module pip not found" generally means something else wasn't found and needs to be added, so it was trying and failing to install it with pip

Comment (Luciogi) [reply srcpkgs/tuimoji/template:9]:
```
=> tuimoji-1.0.0_8: running do_check ...
running test
WARNING: Testing via this command is deprecated and will be removed in a future version. Users looking for a generic test entry point independent of test runner are encouraged to use tox.
/usr/lib/python3.12/site-packages/setuptools/command/test.py:193: _DeprecatedInstaller: setuptools.installer and fetch_build_eggs are deprecated.
!!

        ********************************************************************************
        Requirements should be satisfied by a PEP 517 installer.
        If you are using pip, you can try `pip install --use-pep517`.
        ********************************************************************************

!!
  ir_d = dist.fetch_build_eggs(dist.install_requires)
/usr/bin/python3: No module named pip
error: Command '['/usr/bin/python3', '-m', 'pip', '--disable-pip-version-check', 'wheel', '--no-deps', '-w', '/tmp/tmpqi9nfp5u', '--quiet', 'urwid']' returned non-zero exit status 1.
=> ERROR: tuimoji-1.0.0_8: do_check: '${make_check_pre} python3 setup.py ${make_check_target} ${make_check_args}' exited with 1
=> ERROR:   in do_check() at common/build-style/python3-module.sh:28

```
it want to install `wheel` using pip, while `checkdepends="python3-wheel'`

Comment (Luciogi) [reply srcpkgs/tuimoji/template:9]: I realized that this project does not have any test

Comment (Luciogi) [reply srcpkgs/tuimoji/template:9]: ok i figured out thanks

Comment (Luciogi) [reply srcpkgs/tuimoji/template]: Just a question: test passes without `python3-wheel` leave a warning of missing `wheel`
so should I be added in `checkdepends=`

File: srcpkgs/tuimoji/template [hunk 2]
```diff
@@ -3,9 +3,10 @@ pkgname=tuimoji
 version=1.0.0
 revision=7
 build_style=python3-module
-pycompile_module="tuimoji"
+#pycompile_module="tuimoji"
```
Comment (ahesford) [inline srcpkgs/tuimoji/template:6]: Don't leave template directives commented out. Remove the directive if it is no longer relevant.

**Review by ahesford — CHANGES_REQUESTED** [review]

File: srcpkgs/tuimoji/template (hunk 2 above)
Comment (Luciogi) [reply srcpkgs/tuimoji/template]: done

**Comment by Luciogi** [comment]
> This isn't actually running tests.
> 
> https://github.com/void-linux/void-packages/actions/runs/6996075093/job/19031580372?pr=46810#step:7:125

actually tests does not exist

**Comment by ahesford** [comment]
> > This isn't actually running tests.
> > https://github.com/void-linux/void-packages/actions/runs/6996075093/job/19031580372?pr=46810#step:7:125
> 
> actually tests does not exist (https://github.com/Fingel/tuimoji)
> 
> Is it fine to use `make_check=no`?

Yes, remove the `checkdepends` and disable tests.

**Review by ahesford — CHANGES_REQUESTED** [review]
The commit message is not accurate, because tests have been disable, not fixed.

**Pull request closed without merging**